File: Wizard Battle Web API/DTOs/Player/PlayerRequest.cs
namespace Wizard_Battle_Web_API.DTOs.Player
{
	public class PlayerRequest
	{
		public int? AccountID { get; set; }

		[Required(ErrorMessage = "* is required")]
		public string PlayerName { get; set; }

		public int IconID { get; set; }

		public uint ExperiencePoints { get; set; }

		public uint KnowledgePoints { get; set; }

		public uint TimeCapsules { get; set; }

		public uint MatchWins { get; set; }

		public uint MatchLosses { get; set; }

		public uint TimePlayedMin { get; set; }
	}
}

File: Wizard Battle Web Client/src/app/_models/Player/PlayerRequest.ts
export interface PlayerRequest {
    accountID: number | null;
    playerName: string;
    iconID: number;
    experiencePoints: number;
    knowledgePoints: number;
    timeCapsules: number;
    matchWins: number;
    matchLosses: number;
    timePlayedMin: number;
    avgDamage: number;
    avgSpellsHit: number;
    spellBookID: number;
}
